### DISCUSSION TITLE
Who's The New Slint UX Designer?

### DISCUSSION THREAD

**Discussion by NigelBreslaw** [slint-ui/slint#6116] [Blog Comments]
Discussion for the blog post: https://slint.dev/blog/who-is-the-new-designer

**Comment by bjorn** [comment]
Great to see you join the Slint team, Nigel!

**Comment by Horbin-Magician** [comment]
Congratulation!

**Comment by adrian5** [comment]
Congrats on joining the Slint team. All the best!

**Comment by algovoid** [comment]
Congratulation!
